client/me/security-checkup/recovery-email.jsx

```diff
@@ -59,25 +59,29 @@ module.exports = React.createClass( {
 						/>
 				</ManageContact>
 		);
-	},
+	}
 
-	haveEmail: function() {
+	haveEmail() {
```

Comment:
I might be wrong but would this not need to use fat arrow syntax for binding (since it refers to 'this')?